Quadriga/src/main/java/edu/asu/spring/quadriga/domain/workspace/ITextFile.java

```diff
@@ -29,5 +29,14 @@
     public abstract String getFileContent();
 
     public abstract void setFileContent(String fileContent);
+    
+    public abstract String getTextFileURI();
+
+    /**
+     * @param URIPrefix
+     * 
+     * Sets the URI corresponding to the TextFile. 
```

Comment:
I can guess that from the method name. Your javadoc should tell me what the method does and what it expects. Also, please review how to structure javadoc.